### PR TITLE
Feature add remove todo

### PR DESCRIPTION
**Pull request by darikmohammed**
# ToDo list: add and remove
> Implement the CRUD method. 

# Features 

- Remove all hardcoded items from the task array. 
- Create a new Js with add, remove and update functions.
- Update the remaining items after deleting. 
- ToDo list saved in the local storage.

### CODE REVIEW

File: src/modules/List.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+export default class List {
+  constructor() {
+    this.todoList = [];
+  }
+
```
Comment (AnselemOdims) [inline src/modules/List.js:5]: - [ ] Good job on the work so far, but please kindly implement a function for editing tasks descriptions

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @darikmohammed , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/List.js (hunk 1 above)
Comment (darikmohammed) [reply src/modules/List.js:5]: Resolved Thanks. Users can edit the tasks by double clicking now.

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @darikmohammed,

YesYouAreGIF (2)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlight
✅ Correct use of Gitflow
✅ No linter error
✅ App runs fine
✅ README is okay

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**